examples/multiple_viewers_.py

```diff
@@ -13,12 +13,13 @@
 
 # add the image
 photographer = data.camera()
-viewer_a = napari.view_image(photographer, name='photographer')
+viewer = napari.Viewer()
```

Comment:
```suggestion
viewer_a = napari.Viewer()
```
This needs to be viewer_a and the other one, as noted below by Carol, needs to be viewer_b